### ISSUE TITLE
Color Contrast failure on Gravity Forms Image Upload delete button

### ISSUE THREAD

**Issue by amberhinds** [wpa11yday/wpaccessibilityday#133]
This color contrast failure is visible after uploading an image to an image upload field. The trash icon is only visible on hover.
https://2025.wpaccessibility.day/organizer-onboarding/

<img width="629" alt="Image" src="https://github.com/user-attachments/assets/3d4e0cb9-bb76-4804-a6df-986418225638" />

On Hover
<img width="665" alt="Image" src="https://github.com/user-attachments/assets/291dde83-5905-4b9a-8ce6-8f987e80a1e1" />
